<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-08-18T23:24:57 IST" name="Default suite" finished-at="2023-08-18T23:27:19 IST" duration-ms="141641">
    <groups>
    </groups>
    <test started-at="2023-08-18T23:24:57 IST" name="Default test" finished-at="2023-08-18T23:27:19 IST" duration-ms="141641">
      <class name="vtiger.LeadTests.CreateLead">
        <test-method is-config="true" signature="bsConfig()[pri:0, instance:vtiger.LeadTests.CreateLead@415b0b49]" started-at="2023-08-18T23:24:57 IST" name="bsConfig" finished-at="2023-08-18T23:24:57 IST" duration-ms="14" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bsConfig -->
        <test-method is-config="true" signature="bcConfig(java.lang.String,org.testng.ITestContext)[pri:0, instance:vtiger.LeadTests.CreateLead@415b0b49]" started-at="2023-08-18T23:24:57 IST" name="bcConfig" finished-at="2023-08-18T23:25:01 IST" duration-ms="3444" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[browser]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@46a0ef6f]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bcConfig -->
        <test-method is-config="true" signature="bmConfig()[pri:0, instance:vtiger.LeadTests.CreateLead@415b0b49]" started-at="2023-08-18T23:25:01 IST" name="bmConfig" finished-at="2023-08-18T23:25:03 IST" duration-ms="1921" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bmConfig -->
        <test-method signature="createLeadTest()[pri:0, instance:vtiger.LeadTests.CreateLead@415b0b49]" started-at="2023-08-18T23:25:03 IST" name="createLeadTest" finished-at="2023-08-18T23:25:07 IST" duration-ms="4313" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertFalse(Assert.java:67)
at org.testng.Assert.assertFalse(Assert.java:77)
at vtiger.LeadTests.CreateLead.createLeadTest(CreateLead.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createLeadTest -->
        <test-method is-config="true" signature="amConfig()[pri:0, instance:vtiger.LeadTests.CreateLead@415b0b49]" started-at="2023-08-18T23:27:17 IST" name="amConfig" finished-at="2023-08-18T23:27:19 IST" duration-ms="1382" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- amConfig -->
        <test-method is-config="true" signature="acConfig()[pri:0, instance:vtiger.LeadTests.CreateLead@415b0b49]" started-at="2023-08-18T23:27:19 IST" name="acConfig" finished-at="2023-08-18T23:27:19 IST" duration-ms="204" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- acConfig -->
        <test-method is-config="true" signature="asConfig()[pri:0, instance:vtiger.LeadTests.CreateLead@415b0b49]" started-at="2023-08-18T23:27:34 IST" name="asConfig" finished-at="2023-08-18T23:27:34 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- asConfig -->
      </class> <!-- vtiger.LeadTests.CreateLead -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
